### ISSUE TITLE
Gutenboarding: Intent Capture isn't accessible

### ISSUE THREAD

**Issue by dubielzyk** [Automattic/wp-calypso#43383]
Currently the tip showing on the intent capture isn't accessible. [See screenshot](https://d.pr/i/aatTlh). Here's a few other things too:

- [x] Text is gray20, but should be gray50
- [x] Font-size should be 14px not 13px
- [x] The icon should size 18px not 14px
- [x] The _Skip for now_ button label should be gray50 too
- [x] The tip should fade in within 3 seconds like it do, but it shouldn't fade out
- [x] Clicking the tip makes it go away. This also shouldn't be the case.

**Comment by yansern** [comment]
> The tip should fade in within 3 seconds like it do, but it shouldn't fade out.

Could you confirm the following behaviour, especially no. 3?

1. The tooltip appears after 3 seconds of delay.
2. When the user enters the site title, the tooltip remains.
3. When the page is refreshed and a pre-existing site title already exist, the tooltip is shown immediately (without any delay or animation).
4. The tooltip never disappears once it appears.

**Comment by dubielzyk** [comment]
1. Correct
2. Incorrect. The tooltip only remains if the field is empty
3. Incorrect. The tooltip only remains if the field is empty
4. Correct. Unless the field is filled.

**Comment by yansern** [comment]
Thanks! So the specs would be:

1. When the field is empty, the tooltip appears after 3 seconds of delay. (Doesn't matter focused or blurred.)
2. When the field is not empty, the tooltip disappears.

**Comment by dubielzyk** [comment]
Correct 😃
